tests/test_transformDiaSourceCatalog.py

```diff
@@ -41,6 +41,7 @@
 class TestTransformDiaSourceCatalogTask(unittest.TestCase):
     def setUp(self):
         self.nSources = 10
+        # Default PSF size for TestDataset gives an 18 pixel source box.
         self.bboxSize = 18
```

Comment:
Can you force the sizes to match, instead of relying on defaults?

Reply:
Unfortunately, it's coming from the default `psfDim` kwarg to `makeEmptyExposure` in TestDataset. I could tie this to the psf bounding box +1, but that doesn't feel right (and I don't think it's always +1; that probably depends on where within the pixel the source is).

Reply:
Could you read the box size from the test dataset once it's created?

Reply:
`exposure.psf.computeBBox()` gives 17: I'm guessing there's code within ap_association that ensures the source boxes are even? It's tied to the default for `TestDataset`, and I doubt that would ever be changed.

Reply:
I've tweaked the comment a bit, though.